torch/fx/experimental/sym_node.py

```diff
@@ -240,6 +240,13 @@ def __str__(self):
         return self.str()
 
     def __repr__(self):
+        return (
+            f"SymNode(expr={self.expr}, shape_env={self.shape_env}, "
+            f"pytype={self.pytype}, hint={self._hint}, constant={self.constant})"
+        )
+
+    def _graph_repr(self) -> str:
```

Comment:
I guess if we're not using SymNode in graph output we don't actually need this function.  Or you can modify `SymInt.__repr__` to use `self.node._graph_repr()` instead of `str(self.node)`